UI-Assignment3/UI-Assignment3.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style3.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>Document</title>
+</head>
+
+<body>
+    <section id="mainPage">
+        <!-- This is header of page-->
+        <section id="headerSection">
+            <!--This is navigation bar-->
+            <section id="headerSectionChild1">
+                <img src="C:\Visual Studio codesss\UI-Assignment3\logo.png" alt="hii" id="logo">
+                <nav id="navId">
+                    <a class="navButton" href="#headerSectionChild2">Home</a>
+                    <a class="navButton" href="#contentFirst">Feature</a>
+                    <a class="navButton" href="#review">Review</a>
+                    <a class="navButton" href="#download">Download</a>
+                </nav>
+                <div id="iconId">
+                    <i class="fa fa-facebook"></i>
+                    <i class="fa fa-instagram"></i>
+                    <i class="fa fa-google-plus"></i>
+                </div>
+            </section>
+            <!--This is content section of header-->
+            <section id="headerSectionChild2">
+                <section id="headerSectionChild2Section1">
+                    <h1 class="headerSectionChild2head">Get the most fun </h1>
+                    <h1 class="headerSectionChild2head">weather app</h1>
+                    <div class="headerSectionChild2button">
+                        <button id="btn1">Download</button>
+                        <button id="btn2">Features</button>
+                    </div>
+                </section>
+                <section id="headerSectionChild2Section2">
+                    <img src="C:\Visual Studio codesss\UI-Assignment3\image\phone1.png" alt="phone image">
+                </section>
+            </section>
+        </section>
+        <!--This is first heading -->
+        <section>
+            <h1 class="one">Perfect Features</h1>
+            <h1 class="two">Only necessary</h1>
+        </section>
+
+        <!--This is content part of cards-->
+        <main id="contentFirst">
+            <i class="far fa-gem"></i>
+            <article>
+                <div class="contentFirstIcon">
+                    <i class="far fa-gem"></i>
+                </div>
+                <section class="section1">
+                    <h1 style="font-size: 2vw;letter-spacing:1px;">Usability</h1>
+                    <p style="font-size: 1.5vw;letter-spacing:1px;" class="para">Sometime the simplest thing are hardest
+                        to find.So we create a new line for everyday life.</p>
+                </section>
+            </article>
+            <article>
+                <section>
+
+                </section>
+                <section class="section1">

```

Comment:
Use a class here instead of inline style as you have used the same style above also